contributors/guide/contributor-cheatsheet/README-es.md

```diff
@@ -0,0 +1,396 @@
+  - [Pruebas](#pruebas)
+  - [Alias de Correo Importantes](#alias-de-correo-importantes)
+  - [Otros Enlaces Útiles](#otros-enlaces-útiles)
+- [Comunicación Efectiva en GitHub](#comunicación-efectiva-en-github)
+  - [Cómo Ser Excelente Entre Sí](#cómo-ser-excelente-entre-sí)
+    - [Ejemplos de Buena/Mala Comunicación](#ejemplos-de-buenamala-comunicación)
+- [Presentar una Contribución](#presentar-una-contribución)
+  - [Firmar el CLA](#firmar-el-cla)
+  - [Abrir y Responder a Issues](#abrir-y-responder-a-issues)
+    - [Crear un Issue](#crear-un-issue)
+    - [Responder a un Issue](#responder-a-un-issue)
+  - [Abrir un Pull Request](#abrir-un-pull-request)
+    - [Crear un Pull Request](#crear-un-pull-request)
+    - [Ejemplo de Descripción de un Pull Request](#ejemplo-de-descripción-de-pull-request)
+    - [Solucionar Problemas de un Pull Request](#solucionar-problemas-de-un-pull-request)
+  - [Labels](#labels)
+- [Trabajar Localmente](#trabajar-localmente)
+  - [Estrategia de Branch](#estrategia-de-branch)
+    - [Agregar Upstream](#agregar-upstream)
+    - [Mantener tu Fork Sincronizado](#mantener-tu-fork-sincronizado)
+    - [Combinar Commits (Squashing Commits)](#squashing-commits)
+   
+---
+## Recursos Útiles
+
+### Empezando
+
+- [Curso para Colaboradores] - **NUEVO** - El curso de E-Learning para colaboradores de Kubernetes!
+- [Guía para Colaboradores] - Guía sobre cómo comenzar a contribuir al Proyecto
+  Kubernetes.
+- [Guía del Desarrollador] - Guía para contribuir código directamente al Proyecto
+  Kubernetes.
+- [Guía de Seguridad y Divulgación] - Guía para informar vulnerabilidades
+  y el proceso de liberación de seguridad.
+
+### SIGs y Otros Grupos
+
+- [Lista Maestra de Grupos][sigs]
+
+### Comunidad
+
+- [Calendario] - Visualiza todos los eventos de la Comunidad de Kubernetes (reuniones de SIG/WG,
+  eventos, etc.).
+- [kubernetes-dev] - La lista de correo de desarrollo de Kubernetes.
+- [Foro de Kubernetes] - Foro oficial de Kubernetes.
+- [Canales de Slack] - Slack oficial de Kubernetes.
+- [Stack Overflow] - Un lugar para hacer preguntas como usuario final de Kubernetes.
+- [Canal de YouTube] - Canal oficial de la comunidad de Kubernetes.
+
+### Flujo de Trabajo
+
+- [Panel de Gubernator] - Visualiza las Solicitudes de Extracción entrantes y salientes que requieren
+  tu atención.
+- [Prow] - Sistema de CI/CD de Kubernetes.
+- [Tide] - Plugin de Prow que administra fusiones y pruebas. [Panel de Tide]
+- [Comandos de Bots] - Comandos utilizados para interactuar con los Bots de Kubernetes (ejemplos:
+- `/cc`, `/lgtm` y `/retest`).
+- [GitHub Labels] - Lista de labels utilizados en todo el Proyecto Kubernetes.
+- [Búsqueda de Código de Kubernetes], mantenida por [@dims].
+
+### Pruebas
+
+- [Prow] - Sistema de CI/CD de Kubernetes.
+- [Test Grid] - Visualiza pruebas históricas y su información asociada.
+- [Panel de Triage] - Agrega problemas similares para una mejor
+  resolución de problemas.
+
```

Comment:
https://go.k8s.io/triage

Reply:
The link is at the bottom of the page as a reference.